### PR TITLE
Add warning below CSS support noting theme changes reset CSS

### LINKED ISSUE #1905: Custom CSS: Note that custom CSS will refresh on theme change
The Custom CSS resets on a theme change, with the assumption that custom CSS intended for one theme may not be best served on a different theme.

It could help user expectation to have a note in the default comment or somewhere else on the Edit CSS page that the contents of the CSS will drop upon theme change.

### PR DESCRIPTION
**Pull request by chaselivingston**
Fixes #1905

### CODE REVIEW

File: modules/custom-css/custom-css.php [hunk 1]
```diff
@@ -802,6 +802,7 @@ static function admin() {
 				<div id="poststuff">
 					<p class="css-support"><?php echo apply_filters( 'safecss_intro_text', __( 'New to CSS? Start with a <a href="http://www.htmldog.com/guides/cssbeginner/">beginner tutorial</a>. Questions?
 		Ask in the <a href="http://wordpress.org/support/forum/themes-and-templates">Themes and Templates forum</a>.', 'jetpack' ) ); ?></p>
+					<p class="css-support"><?php echo __( 'Note: Custom CSS changes will reset when changing themes.', 'jetpack' ); ?></p>
```
Comment (samhotchkiss) [inline modules/custom-css/custom-css.php]: Something about this wording feels awkward.

Maybe "Custom CSS will be reset when changing themes." might be better?  Thoughts?

**Comment by chaselivingston** [comment]
@samhotchkiss Works for me! I've updated the wording per your suggestion.

**Comment by samhotchkiss** [comment]
LGTM!

**Comment by dereksmart** [comment]
:+1:

**Pull request merged**